crates/dw_apb_uart/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+//! Definitions for snps,dw-apb-uart serial.
+//! Uart snps,dw-apb-uart driver in Rust for BST A1000b FADA board.
+#![no_std]
+
+pub mod dw_apb_uart;
```

Comment:
It is no need to define a new mod `dw_apb_uart`, move codes from `dw_apb_uart.rs` to `lib.rs`.